content/docs/dvclive/api-reference/live/log_image.md

```diff
@@ -0,0 +1,56 @@
+# Live.log_image()
+
+```py
+def log_image(name: str, val):
+```
+
+#### Usage:
+
+```py
+from dvclive import Live
+
+live = Live()
+
+import numpy as np
+img_numpy = np.ones((500, 500), np.uint8) * 255
+live.log_image("numpy.png", img_numpy)
+
+from PIL import Image
+img_pil = Image.new("RGB", (500, 500), (250, 250, 250))
+live.log_image("pil.png", img_pil)
+```
+
+## Description
+
+Saves the given image `val` to the output file `name`.
+
+Supported values for `val` are:
+
+- A valid numpy array (convertable to image via `PIL.Image.fromarray`)
```

Comment:
Are there numpy arrays that aren't convertible to `PIL.Image.fromarray`?

Reply:
Those with wrong shape and/or type

Reply:
Do you think a link to https://pillow.readthedocs.io/en/stable/reference/Image.html#PIL.Image.fromarray is worthwhile (just a suggestion, no strong opinion)?